doc/concepts/embeddings.rst

```diff
@@ -0,0 +1,45 @@
+
+Let's consider classical input data consisting of :math:`M` examples, with :math:`N` dimensions each: :math:`\mathcal{D}=\{x^{1}, \ldots, x^{M}\}`. To embed this data into a quantum system (n qubits *or* n qumodes for discrete and continuous variable quantum computing, respectively), we can use various embedding techniques, some of which are explained briefly below. 
+
+
+Basis Embedding
+^^^^^^^^^^^^^^^^^^^^
+
+For the basis embedding, classical data has to be in the form of binary strings. The embedded quantum state is the bit-wise translation of a binary string to the corresponding states of quantum subsystems. For example, :math:`x=1001` is represented by the 4-qubit quantum state :math:`\mid 1001 \rangle`. Hence, one bit of classical information is represented by one quantum subsystem.
+
+Let's consider the classical dataset :math:`\mathcal{D}` mentioned above. For basis embedding, each example has to be a N-bit binary string :math:`x^{m}=(b_1,\ldots,b_N)` with :math:`b_i \in \{0,1\}` for :math:`i=1,\ldots,N`. Assuming all features are repesented with unit binary precision (one bit), each input example :math:`x^{m}` can be directly mapped to :math:`\mid x^{m}\rangle` quantum state. **This means that the number of quantum subsystems, n, must be at least equal to N; N features are represented by N qubits.** The input data can be represented in the computational basis as
+
+.. math:: \mid \mathcal{D} \rangle = \frac{1}{\sqrt{M}} \sum_{m=1}^{M} \mid x^{m} \rangle.
+
+For example, let's say we have a classical data set containing two examples :math:`x^{1}=01` and :math:`x^{2}=11`. The corresponding basis encoding uses two qubits to represent :math:`\mid x^{1} \rangle=\mid 01 \rangle` and :math:`\mid x^{2} \rangle=\mid 11 \rangle`: 
+
+.. math:: \mid \mathcal{D} \rangle = (0) \mid 00 \rangle+\frac{1}{\sqrt{2}}\mid 01 \rangle+(0) \mid 10 \rangle+ \frac{1}{\sqrt{2}} \mid 11 \rangle
+
+.. note:: For :math:`N` bits, there are :math:`2^N` possible basis states. Given :math:`M \ll 2^N`, the basis embedding of :math:`\mathcal{D}` will be sparse :cite:`Mariabook2018`. 
+
+
+Amplitude Embedding
+^^^^^^^^^^^^^^^^^^^^
+This is a popular embedding technique where data is encoded into the amplitudes of a quantum state. This can be easily understood if we concatenate all the input examples :math:`x^m` together into one vector, i.e., 
```

Comment:
I would not judge that it is "popular", things change so quickly. :)

Reply:
yeah. I will remove it .